test_runner/src/main/kotlin/ftl/gc/GcStorage.kt

```diff
@@ -101,6 +103,14 @@ object GcStorage {
         )
     }
 
+    fun IArgs.uploadSessionId() = takeUnless { disableResultsUpload }?.let {
+        upload(
+            file = Path.of(localResultDir, SESSION_ID_FILE).toString(),
```

Comment:
We use `Paths` in other places in the project. Your implementation is not wrong! I am just wondering if it is possible to use `Paths` here as well

Reply:
Changed! Thanks! 👍